### DISCUSSION TITLE
Upgrading team to pro failing

### DISCUSSION THREAD

**Discussion by AStox** [vercel/community#5371] [Account Issue] [answered]
### Question

Hi, I'm trying to upgrade my team Ad corp to the pro tier, but consistently getting the pictured error saying "Failed to upgrade team. Please try again".

<img width="508" alt="image" src="https://github.com/vercel/community/assets/6720265/f92e2de8-7155-4339-ac29-88cc6ec8ebe5">

**Comment by leerob** [comment]
Hey there, sorry about this – would you mind opening a support ticket or share more information here about your account?

**Comment by AStox** [comment]
Sure, any information in particular that would help?
Team name is "Ad Corp" (https://vercel.com/adcorp), currently in the hobby tier, with one project. I'm the only member on it. It has a valid credit card attached. The account is not new but the payment method was added today.

**Reply by calebboyd** [reply, marked as answer]
Hey @AStox We found the source of the issue here! Thanks for your report. You should be able to upgrade now.

**Reply by AStox** [reply]
Upgraded successfully! Thanks for the help!
